server/gc/src/main/java/org/apache/accumulo/gc/metrics2/GcHadoopMetrics2.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.gc.metrics2;
+
+import static org.apache.accumulo.gc.metrics2.GcMetrics.GC_METRIC_PREFIX;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.accumulo.core.gc.thrift.GcCycleStats;
+import org.apache.hadoop.metrics2.lib.MetricsRegistry;
+import org.apache.hadoop.metrics2.lib.MutableGaugeLong;
+
+/**
+ * Hadoop metrics2 implementation of Accumulo GC cycle metrics.
+ */
+class GcHadoopMetrics2 {
```

Comment:
This separate class looks like a holdover from when the legacy metrics and the Hadoop Metrics2 implementations were split. Since we dropped the legacy metrics, there is only the Metrics2, so this can be merged into the GcMetrics class, following the same pattern as all the other metrics implementations.